tests/integrationTests/redirectToOriginalUrl.test.js

```diff
@@ -0,0 +1,74 @@
+import { beforeEach, afterAll, beforeAll, describe, it, expect } from 'vitest';
+import { createServer } from '../../src/server.js';
+
+describe('integration redirect to original url', () => {
+    let server;
+    let shortCode;
+    const originalUrl = 'https://www.pcdf.df.gov.br/servicos/delegacia-eletronica';
+
+    beforeAll(async () => {
+        server = createServer();
+        await server.ready();
+    });
+
+    afterAll(async () => {
+        await server.close();
+    });
+
+    beforeEach(async () => {
+        const reponse = await server.inject({
+            method: 'POST',
+            url: '/',
+            payload: { url: originalUrl }
+        });
+
+        const body = JSON.parse(reponse.body);
+        const shortUrl = body.shortUrl;
+        shortCode = shortUrl.split('/').pop();
+    });

```

Comment:
Isso vai fazer com que antes de cada requisição você rode essa request novamente, mesmo quando ela não é necessária, como no teste de `'should return error when short code does not exist'`. Prefira agrupar o setup dos testes onde somente eles serão utilizados pra evitar rodá-las desnecessariamente